app/controllers/api/databases.php

```diff
@@ -2330,11 +2333,38 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
         $data['$id'] = $document->getId();                      // Make sure user doesn't switch document unique ID
         $data['$permissions'] = $permissions;
 
+        $timestamp = null;
+        $timestampHeader = $request->getHeader('x-appwrite-timestamp');
+        if (!empty($timestampHeader)) {
+            $timestamp = new \DateTime($timestampHeader);
+        }
+
         try {
+            $internalCollectionId = 'database_' . $database->getInternalId() . '_collection_' . $collection->getInternalId();
```

Comment:
The name `$internalCollectionId` is a little confusing since it does not equal `$collection->getInternalId()`

Reply:
@abnegate do you have a recommendation for a different variable name?

Reply:
@abnegate, I've updated the variable name to `$databaseCollectionId`.

Reply:
I think `$privateCollectionId` might be a bit clearer